libs/brig-types/src/Brig/Types/Intra.hs

```diff
@@ -116,6 +117,16 @@ instance ToJSON UserAccount where
       other ->
         error $ "toJSON UserAccount: not an object: " <> show (encode other)
 
+newtype LocaleRsp = LocaleRsp {fromLocaleRsp :: Locale}
```

Comment:
```suggestion
newtype LocaleResponse = LocaleResponse {fromLocaleResponse :: Locale}
```

I'm suggesting this because it took me a moment to guess that, and I'm still not a 100% certain.

Reply:
also, given that we want to create swagger for internal end-points as well, and just for consistency: can you use schema-profunctor to generate the json instances?